worker/docker-entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Same as for backend

Reply:
I actually used this because I assumed that in production, some environment variables might be set using docker secrets and this script would fetch the contents of the secret. 
For example, the db password might be set as:
```
    environment:
      - DB_PASSWORD_FILE=/run/secrets/db_password
 ```

Reply:
We don't use those in production ; we store them in k8s secrets and pass them as env directly